fdbserver/SkipList.cpp

```diff
@@ -1052,11 +1052,11 @@ class MiniConflictSet : NonCopyable {
 	}
 	wordType highBits(int b){ // bits (b&bucketMask) and higher are 1
 		#pragma warning(disable: 4146)
-		return -(wordType(1) << b);
+		return -(wordType(1) << (b & bucketMask));
```

Comment:
If you wanted, you could write this as the following (maybe also changing the return type of `bitMask` to `wordType`). I'll leave it up to you whether you think that is better or worse.

```suggestion
		return -bitMask(b);
```